specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/loadBalancer.json

```diff
@@ -1461,6 +1461,79 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.Network/loadBalancers/{loadBalancerName}/loadBalancingRules/{loadBalancingRuleName}/health": {
+      "post": {
+        "tags": [
+          "LoadBalancers"
+        ],
+        "operationId": "LoadBalancerLoadBalancingRules_Health",
+        "description": "Get health details of a load balancing rule.",
+        "parameters": [
+          {
+            "name": "groupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group.",
+            "pattern": "^[a-z][a-z0-9]*$"
+          },
```

Comment:
RG type should be defined in common-types.

Reply:
I don't think we have a common-types json for network, and I can see other APIs use the same format instead of a common type. Is it really necessary? And if so, is solution creating a new common-types json file in that folder?